### ISSUE TITLE
Unsupported Datatype error when using Partitioning

### ISSUE THREAD

**Issue by sandeep-eroad** [demetrixbio/FSharp.Data.Npgsql#50]
I am getting Unsupported Datattype error when using Partitioning. Here is the script to create the tables

```
create table logs (
    id uuid not null,
    log_time timestamp not null
) partition by range (log_time);

create table logs_2019 partition of  logs
for values from ('2019-01-01') to ('2019-12-31');

create table logs_2020 partition of logs
for values from ('2020-01-01') to ('2020-12-31');

select * from logs where log_time between '2019-01-01' and '2019-12-31';
select * from logs_2019 where log_time between '2019-01-01' and '2019-12-31';
```

Now try accessing this using Type-Provider to see the error.

**Comment by NatElkins** [comment]
https://github.com/demetrixbio/FSharp.Data.Npgsql/issues/46 may be relevant

**Comment by daz10000** [comment]
We probably need to extend to handle these edge cases.  Would be helpful to get a (failing) unit test we can use for reproduction.  We unfortunately aren’t hitting these issues but I would like to clean them up because I’m sure we will hit them eventually :)

Von meinem iPhone gesendet

**Comment by kerams** [comment]
@sandeep-eroad What exact command and Postgres version are you using? `select * from logs_2019` compiles for me on 12.1. `select * from logs` doesn't, because column schema is not retrieved for partitioned tables (https://github.com/demetrixbio/FSharp.Data.Npgsql/blob/master/src/DesignTime/InformationSchema.fs#L409). It's a quick fix, but #56 should also sort of help on its own.

**Comment by sandeep-eroad** [comment]
I used version PG 11.2.
